src/query/execute_statement.c

```diff
@@ -15348,12 +15348,12 @@ do_supplemental_statement (PARSER_CONTEXT * parser, PT_NODE * statement, RESERVE
 
 	    if (cls_info[i]->objtype == CDC_TABLE)
 	      {
-		strncpy (drop_stmt, drop_prefix, strlen (drop_prefix));
+		strncpy (drop_stmt, drop_prefix, strlen (drop_prefix) + 1);
```

Comment:
strncpy(dst, src, strlen(src) + 1) is equivalent to strcpy(dst, src) and can be simplified.

Reply:
I habitually use strncpy inside this function, but it would be better to replace it with strcpy.
If we go a little further,
A more desirable solution is possible by changing the order of the process of finding the length after copying.


```
strncpy (drop_stmt, drop_prefix, strlen (drop_prefix) + 1);
drop_copied_length = strlen (drop_prefix);

```
Replace the form above as below (there is a small gain in performance)
```
drop_copied_length = strlen (drop_prefix);
memcpy (drop_stmt, drop_prefix, len + 1);
```